src/main/java/edu/kis/powp/jobs2d/commands/ComplexCommand.java

```diff
@@ -0,0 +1,28 @@
+package edu.kis.powp.jobs2d.commands;
+
+import java.util.ArrayList;
+
+public class ComplexCommand implements DriverCommand {
+
+    private ArrayList<DriverCommand> commands = new ArrayList<>();
```

Comment:
List type (attribute and in the constructor) would be sufficient and would not limit the implementation.